src/Farmer/Builders/Builders.WebApp.fs

```diff
@@ -483,21 +483,33 @@ type WebAppConfig =
                 { site with AppSettings = None; ConnectionStrings = None } // Don't deploy production slot settings as they could cause an app restart
                 for (_,slot) in this.CommonWebConfig.Slots |> Map.toSeq do
                     slot.ToSite site
-                
+            
+            let mutable dependentCustomDomainId = Option<ResourceId>.None
```

Comment:
You shouldn't need to specify `Option<ResourceId>` just specifying `None` should be sufficient here